frontend/src/features/settings/pages/SpecialAuthorizations/LOA/LOASteps.tsx

```diff
@@ -77,7 +79,8 @@ export const LOASteps = ({
 
   const handleFinish = async () => {
     // Handle submitting LOA
-    const res = !loaId ? await createLOAMutation.mutateAsync({
+    const isLOACreation = !loaId;
```

Comment:
Is this just a readability change?

Reply:
Not just a readability change, since a notification snackbar needs to be shown after the LOA creation/update endpoint has been requested successfully, and this "isLOACreation" variable can be used to determine whether or not the LOA action was creation or update, since a success snackbar is shown for creation and an info snackbar is shown for update

Reply:
I could use ```!loaId``` for each of the ternary assignments to determine creation or update, but perhaps reusing this isLOACreation variable might be a better way of doing it